lucid/modelzoo/vision_base.py

```diff
@@ -87,21 +87,15 @@ def _layers_from_list_of_dicts(model_class, list_of_dicts):
   return tuple(layers)
 
 
-class ModelPropertiesMetaClass(type):
-
-  @property
-  def name(cls):
-      return cls.__name__
-
-
-class Model(with_metaclass(ModelPropertiesMetaClass, object)):
+class Model():
   """Model allows using pre-trained models."""
 
   model_path = None
   labels_path = None
   image_value_range = (-1, 1)
   image_shape = (None, None, 3)
   layers = ()
+  modelname = None
```

Comment:
maybe this should be named `model_name` to be more consistent with the rest of naming?

Reply:
Agree, fixed